app/controllers/cards_controller.rb

```diff
@@ -34,10 +34,26 @@ def update
     end
   end
 
+  def show_random_card
+    @card = Card.random_need_to_review
+    render 'random_card'
+  end
+
+  def check
+    if @card.original_text == params[:card_translate][:original_text]
+      flash[:notice] = "Правильно"
+      @card.update_review_date
+      redirect_to root_path
+    else
+      flash.now[:error] = "Не правильно"
+      render 'random_card'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.